cmd/convox/stdcli/stdcli.go

```diff
@@ -115,6 +115,23 @@ func Debug() bool {
 	return false
 }
 
+// RecoverFlag allows us to capture things like --app FOO which would otherwise be discarded by urfave/cli if passed in position 0
+func RecoverFlag(c *cli.Context, flagNames []string) string {
```

Comment:
We could change the flags to a variadic variable:

`RecoverFlag(c *cli.Context, flags ...string) string`

This way when calling the function, we can `RecoverFlag(c, "a", "app")`